delphi-frontend/src/main/java/au/com/integradev/delphi/msbuild/DelphiProjectParser.java

```diff
@@ -81,31 +83,29 @@ private static Set<String> createUnitScopeNames(ProjectProperties properties) {
   }
 
   private List<Path> createSearchDirectories(Path dprojDirectory, ProjectProperties properties) {
-    /*
-     We manually append the library paths here, even though it's not strictly correct to do so.
+    List<Path> allPaths = new ArrayList<>();
```

Comment:
I don't think `allPaths` is the right name for this variable anymore. Just `paths` is probably OK? or `unitSearchPaths`?